spec/features/items/user_views_a_item_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../shared_scenarios'
+
+RSpec.feature 'user views a item' do
```

Comment:
`user views an item` reads better, and is a more correct sentence, no?

Reply:
Fixed